### DISCUSSION TITLE
Should we have a breaking changelog?

### DISCUSSION THREAD

**Discussion by luong-komorebi** [LunarVim/LunarVim#2023] [Q&A] [answered]
When I look at the rolling branch right now, I am a bit worried about all the changes that might come in the future when we merge rolling to master.

For me as user, I have to watch each commit and notice what changed. I am not sure if there weree any other alternatives. Reading the docs might be one, but it had other concerns like docs are not up to date. 

I am looking for a way to make this easier, especially for people who once in a while update their lunarvim.

I have seen other communities organising an issue where they tried to notify users of what will break in the next version and we follow the comments there to change our configuration. 
https://github.com/neovim/neovim/issues/14090 

I wonder what maintainers would think about this and how we as users can help maintainers turn this idea into reality

**Comment by kylo252** [comment, marked as answer]
We don't really have that many of those, and even then, I don't usually do any breaking changes without documenting them in the README first.

That said, I still miss some from time to time, so maybe a pinned issue is a good idea after all.

**Reply by lcptrs** [reply]
I'd definitely like to see breaking changes list in the release. For example I updated yesterday and suddenly jj -> \<Esc\> didn't work. Turned out, it was removed as a "small improvement" with https://github.com/LunarVim/LunarVim/pull/3021 which is certainly true for many, but for me personally it broke a fundamental part of my workflow.

**Reply by milind-shakya-sp** [reply]
Yup, as a user of LunarVim, it would be nice to receive notifications of breaking changes. Last thing we want is to impact the already  awesome user experience in a negative way by introducing these random changes.
